dig.lua

```diff
@@ -0,0 +1,50 @@
+-- digs a square channel around the tile selected (use: dig x y z, dig 3 3 3 -> 7x7x3 channel)
```

Comment:
This script actually conflicts with the `dig` plugin - although that plugin doesn't implement a `dig` command, any links in the documentation are ambiguous:
```
scripts/dig.lua:4: WARNING: duplicate label dig, other instance in dfhack/docs/Plugins.rst
```

Reply:
Indeed. I'm planning write a channel-safety plugin. Something to manage channel designations safely for the dumb dorfs.. to prevent them from digging from the bottom up - something my dwarves seem really keen on doing.

Reply:
I spotted a couple more UX-related issues with this one, so I'm thinking of leaving it out for now, since they're probably non-trivial to address:
- It doesn't generate floors
- It doesn't reveal tiles that it cleared, so it's a bit hard to tell what it did if you're using it in an unrevealed area

I would definitely be interested in seeing what sort of channel-safety tool you come up with, though!